common/src/main/java/io/homeassistant/companion/android/common/data/integration/Entity.kt

```diff
@@ -649,6 +649,15 @@ fun <T> Entity<T>.friendlyState(context: Context, options: EntityRegistryOptions
             }
         }
     }
+
+    if (appendUnitOfMeasurement) {
+        val unit = (attributes as? Map<*, *>)?.get("unit_of_measurement")?.toString()
+
+        if (unit?.isNotEmpty() == true) {
```

Comment:
Because this is for a String, use `isNotBlank()` instead.

Reply:
Makes sense, fixed :+1: